src/candela/VisComponent/README.md

```diff
@@ -5,37 +5,33 @@ The base class for Candela visualization components.
 This class is intentionally minimal, because there are only a few common
 features of all Candela components:
 
-1. Candela components work on the web, so the constructor looks like ``new
-   VisComponent(el)``, where ``el`` is (usually) a DOM element. The
-   ``VisComponent`` constructor attaches ``el`` to the object, so you can always
-   refer to it using ``this.el``.
+1. Candela components work on the web, so the constructor looks like `new
+   VisComponent(el)`, where `el` is (usually) a DOM element. The
+   `VisComponent` constructor attaches `el` to the object, so you can always
+   refer to it using `this.el`.
 
 2. Candela components perform some type of visualization, so they have a
-   ``render()`` method. The ``VisComponent`` ``render()`` method simply throws
-   an exception; if you truly want your component to do nothing when it renders,
-   simply redefine the method to be a no-op.
+   [`render()`](#componentrender) method. The base class [`render()`](#componentrender)
+   is not implemented and raises an exception.
```

Comment:
Let's omit "is not implemented and", since technically the method *is* implemented.